Cyra/lab2/part2/include/hello.hpp

```diff
@@ -0,0 +1,6 @@
+class Hello
```

Comment:
looks like the header guard didnt get commit-ed :shrug:

Reply:
Woops, just fixed it